torch/csrc/jit/ir/constants.cpp

```diff
@@ -88,8 +88,8 @@ c10::optional<Value*> tryInsertConstant(
     n->i_(attr::value, val.toBool());
     n->output()->setType(BoolType::get());
   } else if (val.isList()) {
-    bool fast_path_list =
-        val.isBoolList() || val.isIntList() || val.isDoubleList();
+    bool fast_path_list = val.isBoolList() || val.isIntList() ||
+        val.isDoubleList() || val.isComplexDoubleList();
```

Comment:
As I recall, complex literals are not supported in TorchScript yet, so this is not being tested.

Reply:
removed!